Calculator/Calculator.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * controller class for the calculator application
+ * 
+ * @author Udit Saxena
+ *
+ */
+
+public class Calculator {
+
+	/**
+	 * perform controlling function of taking input pass it for evaluating
+	 * display the result
+	 * 
+	 */
+	public void actionPerformed() {
```

Comment:
actionPerformed stating that some action is already performed, this can be performAction instead